src/address.rs

```diff
@@ -769,4 +795,43 @@ mod test {
             roundtrips(&addr);
         }
     }
+
+    #[test]
+    fn test_blech32_vectors() {
+        // taken from Elements test/functional/rpc_invalid_address_message.py
```

Comment:
Is there a reason why there is no `BLECH32M_VALID` test vector?

Reply:
There is also no `BECH32M_VALID` test vector. I simply converted all the existing Bitcoin test vectors (which exist to test the RPC error messages as much as they exist to test the decoding logic).